### DISCUSSION TITLE
Seeking Assistance: Locating Git LFS Settings in GitHub Repository

### DISCUSSION THREAD

**Discussion by vv01sam** [community/community#101765] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm currently managing a GitHub repository and I've been trying to confirm if Git Large File Storage (LFS) is enabled. However, I am unable to locate any specific section related to Git LFS in the repository's settings. Usually, this setting might be found under categories related to files or storage, but it doesn’t seem to be present. Could someone guide me on how to check for Git LFS support in the repository settings, or provide insight if this feature is managed differently?

**Comment by fury-05** [comment]
Check .gitattributes file: Open your repository on GitHub, navigate to the root of the repository, and check if there is a file named .gitattributes. This file is used to specify how Git should handle certain files, and it's commonly used to configure Git LFS.

If you have a .gitattributes file, look for lines that specify Git LFS tracking. They usually look like this:
```yaml
*.jpg filter=lfs diff=lfs merge=lfs -text
```
This line, for example, indicates that files with the .jpg extension should be tracked using Git LFS.
